Makefile

```diff
@@ -2,8 +2,8 @@ SHELL := /bin/sh
 APPNAME = autopush
 DEPS =
 HERE = $(shell pwd)
-PTYPE=python
-BIN = $(HERE)/bin
+PTYPE=pypy
```

Comment:
I wonder if we should eventually add a task that sets up a CPython virtualenv and symlinks `python` to `pypy` for local development.